src/component/Footer.js

```diff
@@ -0,0 +1,35 @@
+﻿import "../style/footer.css";
+import facebookIcon from "../image/facebook.svg";
+import twitterIcon from "../image/twitter.svg";
+import youtubeIcon from "../image/youtube.svg";
+import instagramIcon from "../image/instagram.svg";
+
+function Footer() {
+  return (
+    <footer>
+      <div class="footer_frame">
+        <div class="maker">©codeit - 2023</div>
+        <div class="sub">
+          <a href="./privacy.html">Privacy Policy</a>
+          <a href="./faq.html">FAQ</a>
+        </div>
+        <div class="icons">
+          <a href="https://www.facebook.com/">
+            <img class="icon" src={facebookIcon} alt="facebook" />
+          </a>
+          <a href="https://twitter.com/home?lang=ko">
+            <img class="icon" src={twitterIcon} alt="twitter" />
+          </a>
+          <a href="https://www.youtube.com/">
+            <img class="icon" src={youtubeIcon} alt="youtube" />
+          </a>
+          <a href="https://www.instagram.com/">
+            <img class="icon" src={instagramIcon} alt="instagram" />
+          </a>
```

Comment:
```suggestion
          {SNS_LIST.map(({ name, href, src }) => {
	              return (
	                <a key={name} href={href}>
	                  <img className="icon" src={src} alt={`${name} icon`} width="20" height="20" />
	                </a>
	              );
	            })}
```

```js
const SNS_LIST = [
  { name: 'facebook', href: 'https://www.facebook.com', src: facebookIcon },
  { name: 'twitter', href: 'https://twitter.com', src: twitterIcon },
  { name: 'youtube', href: 'https://youtube.com', src: youtubeIcon },
  { name: 'instagram', href: 'https://www.instagram.com', src: instagramIcon },
];
```
이런식으로 배열을 구성한 뒤 map 돌리는 로직으로 구현하게 되면 추후 SNS 아이콘이 추가/수정/삭제될때 대응하기 편해집니다.